files/start.sh

```diff
@@ -4,27 +4,31 @@
 rm -rf /app/tmp/*
```

Comment:
Probably belongs to a separate PR, but I'm looking for an approach similar to HN2 instead of this file.